README.md

```diff
@@ -152,3 +142,18 @@ Workers Sites with Wrangler bundles up a text blob that maps request paths to co
 let assetManifest = { "index.html": "index.special.html" }
 return getAssetFromKV(event, { ASSET_MANIFEST: JSON.stringify(assetManifest) })
 ```
+
+## Other functions
+
+#### `mapRequestToAsset`
+
+type: function(Request) => Request
+
+The default function for mapping incoming requests to keys in Cloudflare's KV.
+
+Takes any path that ends in `/` or evaluates to an html file and appends `index.html` or `/index.html` for lookup in your Workers KV namespace. This works for most static site generators, but you can customize this behavior by passing your own function as `mapRequestToAsset`. The function should take a `Request` object as its only argument, and return a string representation of the path to be looked up in the asset manifest/KV.
+
+### `serveSinglePageApp`
+type: function(Request) => Request
+
+A custom handler for mapping requests to a single root: `index.html`. The most common use case is single-page applications - frameworks with in-app routing - such as React Router, VueJS, etc. It takes zero arguments.
```

Comment:
let's retain an example usage to make clear this is for use as the `mapRequestToAsset` option